howtos/process/bug-tracking.rst

```diff
@@ -0,0 +1,177 @@
+
+.. |label-invalid| image:: images/label-invalid.png
+        :scale: 70
+
+.. |label-duplicate| image:: images/label-duplicate.png
+        :scale: 70
+
+.. |label-won't-fix| image:: images/label-will-not-fix.png
+        :scale: 70
+
+Label |label-verified| is only added by the **bug scrub owner** after
+reviewing the solution and feedback from QA and bug reporter.
+
+.. |label-verified| image:: images/label-verified.png
+        :scale: 70
+
+
+Priority Labels
+-----------------
+**Bug scrub owner** should use them to set priority to a bug according
+to its impact.
+
+.. image:: images/label-priorities.png
+        :scale: 50
+
+
+Plaform and Branch Labels
+----------------------------
+Used by **QA** and **bug reporter**.
+
+*Platform* labels are used to specify a platform or multiple platforms on
+which a bug is observed, e.g. |label-byt|, |label-apl|, |label-glk| ...
+
+.. |label-byt| image:: images/label-platform-byt.png
+        :scale: 70
+
+.. |label-apl| image:: images/label-platform-apl.png
+        :scale: 70
+
+.. |label-glk| image:: images/label-platform-glk.png
+        :scale: 70
+
+*Branch* labels are used specify a branch or multiple branches on which
+a bug is observed, e.g. |label-branch-v1.2|, |label-branch-glk|,
+|label-branch-master| ...
+
+
+.. |label-branch-v1.2| image:: images/label-branch-v1-2.png
+        :scale: 70
+
+.. |label-branch-glk| image:: images/label-branch-glk.png
+        :scale: 70
+
+.. |label-branch-master| image:: images/label-branch-master.png
+        :scale: 70
+
+.. note::
+    *Platform* labels should always be applied.
+
+    *Branch* labels are usually only applied when the branch is not
+    the default branch for developing/release on the platform.
+
+    **QA** should *update (add/remove)* platform and branch labels
+    according to the latest bug status.
+
+Other optional Labels
+-----------------------
+
+Two optional labels can be used to call for attention.
+
+* |label-blocked| - Blocked by some dependency, whichh applies to either
+  feature implementation or bug reproduction.
+
+* |label-need-info| - Further information is requested.
+
+.. |label-blocked| image:: images/label-blocked.png
+        :scale: 70
+
+.. |label-need-info| image:: images/label-need-info.png
+        :scale: 70
+
+How to Report a Bug
+********************
+Please
+`create a issue <https://help.github.com/articles/creating-an-issue/>`_
+and add label |label-bug|.
+
+Please provide the following information:
+
+* A clear bug title that is a concise description of the bug.
+
+* Precise steps to reproduce the bug, and the possibility if you can
+  only reproduce it occasionally.
+
+* Branch name and commit hash of 3 repositories: sof (firmware),
+  linux (kernel driver) and soft (tools & topology).
+
+* Name of the topology file
+
+* Name of the platform(s) on which the bug is observed.
+
+* Kernel message and firmware logger data
+    * Please paste the relevant dmesg and logger data to the comment box.
```

Comment:
They should use markdown for formatting example commands, code, diffs, patches etc. Can you state this and link to markdown guide.

Reply:
Okay. Updated the PR.